source/appModuleHandler.py

```diff
@@ -93,8 +94,7 @@ def getAppNameFromProcessID(processID,includeExt=False):
 	# Try querying the app module for the name of the app being hosted.
 	try:
 		# Python 2.x can't properly handle unicode module names, so convert them.
-		mod = __import__("appModules.%s" % appName.encode("mbcs"),

```

Comment:
This is definitely going to fail on python 3

```
>>> "hello, %s" % "test".encode("mbcs")
"hello, b'test'"
```

str.format gives the same result.

It should be sufficient to check if not isinstance(appName, str) before encoding to mbcs.